packaging/scripts/install.sh

```diff
@@ -60,7 +60,7 @@ install_suse_like() {
 
   # Install testing tools
   zypper install -y --no-recommends \
-    curl gnu-netcat jq
+    curl netcat-openbsd jq
```

Comment:
@Ferroin Out of curiosity, why is this change required?

Reply:
OpenSUSE has apparently stopped shipping the GNU version of netcat for some reason. We end up using netcat when checking the packages as the last part of the CI checks (and as a final layer of insurance before uploading them during nightly builds and release builds), so we need a working copy of netcat.